hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/processors/ProcessorModuleTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.contract.impl.test.exec.processors;
+
+import static com.hedera.node.app.service.contract.impl.exec.systemcontracts.PrngSystemContract.PRNG_PRECOMPILE_ADDRESS;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.hedera.node.app.service.contract.impl.exec.processors.ProcessorModule;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.gascalculator.GasCalculator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class ProcessorModuleTest {
+    @Mock
+    GasCalculator gasCalculator;
+
+    @Test
+    void provideHederaSystemContracts() {
```

Comment:
Do you think this one could use a comment that says "every time you add a new system contract you have to come here and fix up this test" - or will it just be obvious when this test breaks?

Reply:
I think the way these tests are structured could use some refactoring but we can do it in a subsequent pr when we add the ContractCallLocal flow.